pdns/pdns_recursor.cc

```diff
@@ -131,6 +131,7 @@ static thread_local uint64_t t_frameStreamServersGeneration;
 
 thread_local std::unique_ptr<MT_t> MT; // the big MTasker
 std::unique_ptr<MemRecursorCache> s_RC;
+std::unique_ptr<NegCache> s_negcache;
```

Comment:
The `s_` prefix is usually for variables static to a file, or static to a class, so perhaps use `g_` instead?

Reply:
Right. I opt for` g_NegCache`  and might rename `s_RC`  then into `g_RecCache` as well.